tock/hours/forms.py

```diff
@@ -36,7 +36,7 @@ class TimecardForm(forms.ModelForm):
 
     class Meta:
         model = Timecard
-        exclude = ['time_spent', 'reporting_period', 'user']
+        exclude = ['time_spent', 'reporting_period', 'user', 'billable_expectation']
```

Comment:
This is out of scope for this review, but we should change this to explicitly include rather than include, per Django recommendations.

Reply:
Adding the recommendation reference here: https://docs.djangoproject.com/en/3.0/topics/forms/modelforms/#selecting-the-fields-to-use

@tbaxter-18f Since this is out of scope, I recommend adding a new issue to be resolved later.  Does that work?  If so, I can add the issue.

Reply:
Works great for me! Thank you!

Reply:
https://github.com/18F/tock/issues/973